stories/core_components/GeoPushpinChart.tsx

```diff
@@ -68,6 +68,20 @@ storiesOf("Core components/GeoPushpinChart", module)
             }),
         ),
     )
+    .add("with location and viewport", () =>
+        screenshotWrap(
+            renderGeoPushpinChart({
+                projectId: "storybook",
+                location: ATTRIBUTE_LOCATION_GEOCHART,
+                config: {
+                    ...DEFAULT_CONFIG,
+                    viewport: {
+                        area: "north_america",
+                    },
+                },
+            }),
+        ),
+    )
```

Comment:
We should have two more cases with same above data
1. with World viewport
2. with 'Include all data' viewport

Reply:
New stories has been added. Thanks :)